packages/rspack/src/config/zod.ts

```diff
@@ -309,6 +309,13 @@ const resolveAlias = z.record(
 );
 export type ResolveAlias = z.infer<typeof resolveAlias>;
 
+const resolveTsconfig = z.strictObject({
+	configFile: z.string(),
```

Comment:
should we add `referencesType` here?

Reply:
`referencesType` is used on the Rust side for supporting the enum `references: ['./a/tsconfig.json', './b.tsconfig.json'] | 'auto' | undefined`. This is needed because napi does not support such an enum https://napi.rs/docs/concepts/enum.